Runtimes/Core/Threading/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# TODO: I think we can build the one that actually matters based on the
+#       platform we're building, rather than relying just on macro definitions
+#       and linking empty objects.
+#       https://github.com/swiftlang/swift/issues/77435
+add_library(swiftThreading OBJECT
+  "${PROJECT_SOURCE_DIR}/../../lib/Threading/C11.cpp"
+  "${PROJECT_SOURCE_DIR}/../../lib/Threading/Linux.cpp"
+  "${PROJECT_SOURCE_DIR}/../../lib/Threading/Pthreads.cpp"
+  "${PROJECT_SOURCE_DIR}/../../lib/Threading/Win32.cpp")
```

Comment:
We can use `SwiftCore_SWIFTC_SOURCE_DIR` here no?

Reply:
I think it is on the most recent commit?